sklearn/multioutput.py

```diff
@@ -794,6 +792,7 @@ class labels for each estimator in the chain.
            [0.0321..., 0.9935..., 0.0625...]])
     """
 
+    @_fit_context(prefer_skip_nested_validation=False)
```

Comment:
Would it be enough to validate the `fit` of `_BaseChain`. It seems that `super.fit()` is call first.